### PR TITLE
Language Picker Changes

### PR DESCRIPTION
**Pull request by blakegreen32**
Ticket: https://nanopay.atlassian.net/browse/NP-2712

Concerned about the country name change if this is being explicitly checked anywhere we would have to update that as well.

Screen Shot 2020-11-16 at 1 19 18 PM

### CODE REVIEW

File: src/foam/nanos/auth/LanguageChoiceView.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ foam.CLASS({
   }
   ^ .popUpDropDown > div {
     margin-left: -16px;
+    margin-right: -16px;
```
Comment (kennyqiyenkan) [inline src/foam/nanos/auth/LanguageChoiceView.js:65]: Is there a particular reason we are using negative margins here?

Comment (blakegreen32) [reply src/foam/nanos/auth/LanguageChoiceView.js:65]: Due to the paddings of the divs I used the negative margins to properly align the inner divs.

**Comment by jlhughes** [comment]
Re: Country - the Country name will be translated (believe this is still in dev) so Brazil will be Brasil for Portuguese, for example. 
@blakegreen32 what was your concern about the country name?

**Comment by blakegreen32** [comment]
> Re: Country - the Country name will be translated (believe this is still in dev) so Brazil will be Brasil for Portuguese, for example.
> @blakegreen32 what was your concern about the country name?

My concern is the change from USA to US. If we are checking somewhere for "USA" it will break

**Pull request merged**